redis-filler/src/main/java/in/ramanujam/redis/filler/RedisFiller.java

```diff
@@ -16,6 +18,22 @@ public static void addBitcoin( BitcoinRecord bitcoin )
     System.out.println( "RedisFiller :: Id = " + bitcoin.getId() + " count = " + ++count  );
   }
 
+  public static void addBitcoins(List<BitcoinRecord> bitcoins )
+  {
+    Jedis jedis = new Jedis( RedisProperties.getInstance().getRedisContainerHost(),
+        RedisProperties.getInstance().getRedisContainerExternalPort());
+    Pipeline pipeline = jedis.pipelined();
+    bitcoins.forEach(
+        b -> {
+          pipeline.hset(RedisProperties.getInstance().getRedisHashsetName(), b.getId().toString(), b.getKey() );
```

Comment:
I mean that `RedisProperties.getInstance().getRedisHashsetName()` in constant and calling it every time litters code